README.md

```diff
@@ -25,117 +25,120 @@ The I2Cwrapper framework consists of **four basic components.** The first two dr
 
-### Error handling
+### Error Handling
 
 If I2C transmission problems occur, any command to the I2C target could fail and every return value could be corrupted. Depending on context, this could lead to severe consequences, e.g. with uncontrolled  stepper motor movements. That's why I2Cwrapper transmits each command and response with a CRC8 checksum. To find out if a controller's command or a target's response was **transmitted correctly**, the controller can check the following: 
 
-- If I2Cwrapper::sentOK is false, the previous function call was not properly transmitted. 
-- If I2Cwrapper::resultOK is false, the data returned from the previous function call is invalid.
+- If `I2Cwrapper::sentOK` is false, the previous function call was not properly transmitted. 
+- If `I2Cwrapper::resultOK` is false, the data returned from the previous function call is invalid.
 
 The library keeps an internal count of the **number of failed transmissions**, i.e. the number of cases that sentOK and resultOK came back false. If the controller doesn't want to check each transmission separately, it can use one of the following methods at the end of a sequence of transmissions (e.g. after setup and configuration of the target, or at the end of some program loop):
 
-* uint16_t I2Cwrapper::sentErrors() - number of false sentOK events
-* uint16_t I2Cwrapper::resultErrors() - number of false resultOK events
-* uint16_t I2Cwrapper::transmissionErrors() - sum of the above
+* `uint16_t I2Cwrapper::sentErrors()` - number of false `sentOK` events
+* `uint16_t I2Cwrapper::resultErrors()` - number of false `resultOK` events
+* `uint16_t I2Cwrapper::transmissionErrors()` - sum of the above
 
 The respective counter(s) will be reset to 0 with each invocation of these methods.
 
-See the [Error_checking.ino](https://github.com/ftjuh/AccelStepperI2C/blob/controller/examples/Error_checking/Error_checking.ino) example for further illustration.
+See the [Error_checking.ino](examples/Error_checking/Error_checking.ino) example for further illustration.
 
-### Interrupt mechanism
+### Interrupt Mechanism
 
 To keep the controller from having to constantly poll the target device for some new event, the controller can use the `I2Cwrapper::setInterruptPin()` function to tell the target to use one if its pins as an **interrupt line**. The target may use it if it wants to inform the controller about some new event. Of course, an additional hardware line connecting this target pin and a free controller pin is needed to use the interrupt mechanism.
```

Comment:
I have the urge to link almost all your bolded terms to the documentation. I'll control myself mostly.

Reply:
If there's a matching heading, it makes perfectly sense. Most probably I haven't been consistent before. So feel free to add the links.